### PR TITLE
Module I block 2 Review your project from trials

### PR DESCRIPTION
**Pull request by d4nQw3rty**
I created this branch because I will perform a code review on a code from a past project.

### CODE REVIEW

**Comment by d4nQw3rty** [comment]
Hello, @d4nQw3rty congratulation! you are close to finishing:smile: but there are some changes required
--I strongly recommend configuring a workflow and a linter tool,  it is important to improve your coding. You can read this links [https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/](url) and [https://docs.github.com/en/actions/using-workflows/about-workflows](url)

**Comment by d4nQw3rty** [comment]
image
Please drop down your link tag to be after your title tag, for better browsing in the search engine.

**Comment by d4nQw3rty** [comment]
image
I recommend you change your css link position after your title tag it will be better for the browser reading your code :smile:

**Comment by d4nQw3rty** [comment]
image
I notice :thinking:  you are missing text in your `<p>` tag complete it if you do, or delete it if you don't need that element in your code :smiley:

**Comment by ichala** [comment]
> image I notice 🤔 you are missing text in your `<p>` tag complete it if you do, or delete it if you don't need that element in your code 😃

- [ ]  kindly try to phrase the comments politely, as suggestions instead of orders.
For example:

This is a polite suggestion: "I think it would be better to use semantic tags"

This reads as an order: "complete it if you do, or delete it "

**Comment by ichala** [comment]
> image I recommend you change your css link position after your title tag it will be better for the browser reading your code 😄

- [ ] kindly make sure you use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments.

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @d4nQw3rty , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- You Have at least 2 comments 👍 
- You Have some good comments 👍 

 ## Required Changes ♻️

- [ ] **_check the comments below _**


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css"> 
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (d4nQw3rty) [inline index.html:8]: Please kindly Change the position of your l `<link> CSS style` place it after your `<title>` tag it is better for the browsing reading your code :smile:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+</head>
+<body>
+    <h1 id="title">Survey-Form</h1>
+    <p id="description"></p>
```
Comment (d4nQw3rty) [inline index.html:12]: I notice you are missing text in the `<p>` tag :thinking: ; try to write the text missing or delete the element if not needed :smiley:  to have a clean code :laughing:

**Review by d4nQw3rty — COMMENTED** [review]
Hello @d4nQw3rty:  
185516448-88b20da1-9751-4565-a00c-50e890dfeb73

There are some changes required, to reach your goal!! :goal_net:  
Please do not hesitate to ask any questions about the review :smile_cat:

**Review by ichala — APPROVED** [review]
Hi @d4nQw3rty ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Good Comments✔️
- Good use of inline comment✔️


 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- [ ] N/A



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**